spec/system/admin/manage_order_detail_spec.rb

```diff
@@ -10,14 +10,17 @@
   let(:reservation) { create(:purchased_reservation, product: instrument) }
   let(:order_detail) { reservation.order_detail }
   let(:order) { reservation.order }
-
+  let(:administrator) { create(:user, :administrator) }
   let(:director) { create(:user, :facility_director, facility: facility) }
+  let(:logged_in_user) {}
```

Comment:
How about 
```suggestion
  let(:logged_in_user) { director }
```
... since that's what most of these specs are based on